.github/workflows/pull-integration-cluster-k3d.yml

```diff
@@ -51,6 +51,12 @@ jobs:
         run: |
           kubectl create namespace kyma-system
           (cd examples/busola-web-components && kubectl kustomize . | kubectl apply -n kyma-system -f - )
+      - name: Install sap-logging extensibility example
+        shell: bash
+        run: |
+          kubectl create namespace sap-cloud-logging-integration
+          kubectl apply -f tests/integration/fixtures/sap-logging-secret.yaml
+          kubectl apply -f https://raw.githubusercontent.com/kyma-project/telemetry-manager/main/docs/user/integration/sap-cloud-logging/kyma-dashboard-configmap.yaml
```

Comment:
Can we have this config map internally? What happens if they change the naming or remove something? 
Also, we only test one entry so we can make this CM shorter.